### PR TITLE
Created support template under docs folder

### PR DESCRIPTION
**Pull request by kannankvs**
##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by kannankvs** [comment]
Was closed by mistake. Re opening it again

**Comment by kannankvs** [comment]
@mingjunzhang2019  Can you please review as early as possible ?

**Comment by jeff-yin** [comment]
Mingjun is on PTO -- @stalabi1 and/or @kerry-meyer  will need to review.

File: docs/SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+You may obtain a copy of the License at
+
+    https://www.gnu.org/licenses/gpl-3.0.txt
+-->
+
+# Support
+
+  * To report any issue, create an issue [here](https://github.com/dell/<repo name>/issues).
+  * If any requirements have not been addressed, then create an issue [here](https://github.com/dell/<repo name>/issues).
+  * To provide feedback to the development team, send an email to **<email_id>**.
```
Comment (jeff-yin) [inline docs/SUPPORT.md:15]: I don't think we should use e-mail to provide/receive feedback. If folks have feedback they should just open an issue. The development team is subscribed to the repositories so we will get notified via e-mail anyway.

File: docs/SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.gnu.org/licenses/gpl-3.0.txt
+-->
+
+# Support
+
+  * To report any issue, create an issue [here](https://github.com/dell/<repo name>/issues).
+  * If any requirements have not been addressed, then create an issue [here](https://github.com/dell/<repo name>/issues).
```
Comment (jeff-yin) [inline docs/SUPPORT.md:14]: Why don't we link this repo directly? i.e., `https://github.com/ansible-collections/dellemc.enterprise_sonic/issues`

File: docs/SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Licensed under the GPL, Version 3.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.gnu.org/licenses/gpl-3.0.txt
+-->
+
+# Support
+
+  * To report any issue, create an issue [here](https://github.com/dell/<repo name>/issues).
```
Comment (jeff-yin) [inline docs/SUPPORT.md:13]: Why don't we link to this repo directly? i.e., `https://github.com/ansible-collections/dellemc.enterprise_sonic/issues`

**Review by jeff-yin — CHANGES_REQUESTED** [review]

**Comment by kannankvs** [comment]
We are unable to commit in this PR due to "head repository deletion" issue. Hence we raised a new [PR#162](https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/162) and copied all the comments from this PR to the new PR. This PR is no more valid and hence we are closing it.

**Pull request closed without merging**